README.md

```diff
@@ -69,3 +75,78 @@ When the Django server is running, you can start the Django shell with:
 Run the back-end test suite with:
 
     docker-compose exec backend pipenv run python manage.py test --settings=donate.settings_test
+
+
+## Basket
+
+[Basket](https://github.com/mozmeao/basket) is a tool run by MoCo to manage newsletter subscriptions and donations. It's listening for messages (JSON) sent to a SQS queue.
+
+### Basket donations queue:
+
+Basket has 4 [donation event types](https://github.com/mozmeao/basket/blob/master/basket/news/management/commands/process_donations_queue.py#L21-L26): 
+
+- `donation`: process a donation and send its data to SFDC,
+- `crm_petition_data`: add petition signature to SFDC,
+- `newsletter_signup_data`: newsletter signup for the foundation site,
+- `DEFAULT`: process a followup Stripe event on a donation.
+
+For this project, we're only using the `donation` event type.
+
+### Donation event type
+
+Example of a donation message sent to Basket:
+
+```
+{ event_type: 'donation',
+  last_name: 'alex',
+  email: 'alex@alex.org',
+  donation_amount: 50,
+  currency: 'usd',
+  created: 1563801762,
+  recurring: false,
+  service: 'stripe',
+  transaction_id: 'ch_1Ez1TSG8Mmx3htnxyShib70n',
+  project: 'mozillafoundation',
+  last_4: '4242',
+  donation_url: 'http://localhost:3000/en-US/',
+  locale: 'en-US',
+  conversion_amount: 50,
+  net_amount: 48.6,
```

Comment:
Looking at what information Braintree [provides for a transaction](https://developers.braintreepayments.com/reference/response/transaction/python), I don't think we will be able to determine the transaction fee (we get the Paypal transaction fee, but not Braintree's own fee for card payments) and therefore the net amount.